zmq-async.opam

```diff
@@ -12,9 +12,6 @@ build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
 depends: [
   "zmq"
   "jbuilder" {build}
-  "ppx_sexp_conv" {build & >= "v0.9.0"}
-  "base" {>= "v0.9.0"}
```

Comment:
Note that we still depend on `Base` (see `Base.Info.of_string` usage)

Reply:
Done.